php/relate/class-relationship.php

```diff
@@ -60,17 +69,20 @@ public function __construct( $post_id ) {
 	 * @return array|null
 	 */
 	public function get_data() {
-		$cache_data = wp_cache_get( $this->post_id, Sync::META_KEYS['cloudinary'] );
+		$key = $this->post_id . $this->context;
```

Comment:
I would consider adding a separator like a dash or something
maybe adding sanitize_key() as well

Reply:
I can do that, but this key is not public facing, so the benefits are questionable. What I'll do, is to ensure that the return for `get_media_context()` is sanitized.